src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -787,3 +789,87 @@ func (it *StringIterator) takeBuf() interface{} {
     return b
 }
 
+
+// BookOperation manages a long-running operation yielding librarypb.Book.
+type BookOperation struct {
+    lro *longrunning.Operation
+}
+
+// BookOperation returns a new BookOperation from a given name.
+// The name must refer to a long-running operation created by the same service.
```

Comment:
How about: "The name must be that of a previously created BookOperation, possibly from a different process."